### PR TITLE
[OGWA-16] As a user, I want to be able to reset my password

### PR DESCRIPTION
**Pull request by Raffael47**
Added:
- Forgot password
- Reset password

### CODE REVIEW

**Review by asc33nzio — COMMENTED** [review]
- Debug log still in codebase
- Overall OK no conflict

File: projects/client/src/views/Reset/components/field.jsx [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+    confirmPassword: Yup.string()
+      .required("Password confirmation is required")
+      .oneOf([Yup.ref("password")], "Passwords must match"),
+  });
+
+  const handleSubmit = async (value) => {
+    try {
+        console.log(value)
+        console.log(token)
+        console.log(`${process.env.REACT_APP_API_BASE_URL}/user/reset`)
```
Comment (gangsararyop) [inline projects/client/src/views/Reset/components/field.jsx:44]: ketinggalan nih

**Review by asc33nzio — COMMENTED** [review]
- debug logs removed OK

**Review by asc33nzio — APPROVED** [review]
- debug logs removed OK

**Pull request merged**